ETL_KD/MLSstandingsKD.ipynb

```diff
@@ -1980,71 +1950,77 @@
-       "5    16  NaN   9-1-7  NaN   7-8-2    2018  \n",
-       "6     8  NaN  10-2-5  NaN   6-7-4    2018  \n",
-       "7    14  NaN  12-1-4  NaN   4-9-4    2018  \n",
-       "8     6  NaN  11-2-4  NaN   4-8-5    2018  \n",
-       "9    10  NaN  13-2-2  NaN   1-9-7    2018  \n",
-       "10   -2  NaN  11-2-4  NaN   3-9-5    2018  \n",
-       "11   -1  NaN   9-6-2  NaN   6-8-3    2018  \n",
-       "12   -3  NaN  11-2-4  NaN  3-11-3    2018  \n",
-       "13    2  NaN   8-5-4  NaN   5-7-5    2018  \n",
-       "14  -13  NaN   7-5-5  NaN   6-8-3    2018  \n",
-       "15   -6  NaN  11-4-2  NaN  3-12-2    2018  \n",
-       "16   -6  NaN   8-4-5  NaN   2-9-6    2018  \n",
-       "17    0  NaN   8-6-3  NaN  2-10-5    2018  \n",
-       "18  -22  NaN  10-6-1  NaN  1-14-2    2018  \n",
-       "19   -5  NaN   8-7-2  NaN  2-11-4    2018  \n",
-       "20  -13  NaN   6-7-4  NaN  2-11-4    2018  \n",
-       "21  -27  NaN   6-8-3  NaN  2-11-4    2018  \n",
-       "22  -31  NaN   6-7-4  NaN  2-15-0    2018  \n",
-       "23  -22  NaN   2-9-6  NaN  2-12-3    2018  "
-      ]
-     },
-     "execution_count": 9,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "df[\"season\"]=2018\n",
-    "df"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 7,
-   "metadata": {},
+       "0   NaN   W-L-T  NaN   W-L-T  season  \n",
+       "#                                     \n",
+       "1   NaN  14-2-1  NaN   8-5-4    2018  \n",
+       "2   NaN  11-2-4  NaN  10-5-2    2018  \n",
+       "3   NaN  10-2-5  NaN   8-6-3    2018  \n",
+       "4   NaN  10-5-2  NaN   8-6-3    2018  \n",
+       "5   NaN   9-1-7  NaN   7-8-2    2018  \n",
+       "6   NaN  10-2-5  NaN   6-7-4    2018  \n",
+       "7   NaN  12-1-4  NaN   4-9-4    2018  \n",
+       "8   NaN  11-2-4  NaN   4-8-5    2018  \n",
+       "9   NaN  13-2-2  NaN   1-9-7    2018  \n",
+       "10  NaN  11-2-4  NaN   3-9-5    2018  \n",
+       "11  NaN   9-6-2  NaN   6-8-3    2018  \n",
+       "12  NaN  11-2-4  NaN  3-11-3    2018  \n",
+       "13  NaN   8-5-4  NaN   5-7-5    2018  \n",
+       "14  NaN   7-5-5  NaN   6-8-3    2018  \n",
+       "15  NaN  11-4-2  NaN  3-12-2    2018  \n",
+       "16  NaN   8-4-5  NaN   2-9-6    2018  \n",
+       "17  NaN   8-6-3  NaN  2-10-5    2018  \n",
+       "18  NaN  10-6-1  NaN  1-14-2    2018  \n",
+       "19  NaN   8-7-2  NaN  2-11-4    2018  \n",
+       "20  NaN   6-7-4  NaN  2-11-4    2018  \n",
+       "21  NaN   6-8-3  NaN  2-11-4    2018  \n",
+       "22  NaN   6-7-4  NaN  2-15-0    2018  \n",
+       "23  NaN   2-9-6  NaN  2-12-3    2018  "
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df[\"season\"]=2018\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+<<<<<<< Updated upstream
+   "execution_count": 7,
+=======
+   "execution_count": 10,
+>>>>>>> Stashed changes
```

Comment:
This one is very simple. You can accept any of these two. Doesn't really matter.